shared/js/language_list.js

```diff
@@ -8,7 +8,7 @@
 
 (function(exports) {
 
-var LOCALES_FILE = '/shared/resources/languages.json';
+var LOCALES_FILE = '../shared/resources/languages.json';
```

Comment:
I guess we can live without this in this patch, but I have the impression this will help to the Keyboard layouts. Let me know if you want me to remove this from this patch.